### DISCUSSION TITLE
Re-organize List

### DISCUSSION THREAD

**Discussion by warengonzaga** [community/community#8645] [Other Feature Feedback, Questions, & Ideas]
I want something to re-organize it not just filter it.

**Comment by brianlovin** [comment]
Agreed, thanks for the suggestion! If you had to say, what are useful ways you'd want to group or reorganize the contents of a list? Are you thinking you'd want to do things manually, or are you picturing other solutions for this?

**Reply by warengonzaga** [reply]
@brianlovin something like drag and drop to organize it.

**Reply by ashemedai** [reply]
Maybe borrowing an idea from TeamCity (and probably originated elsewhere) might make sense for an initial solution:

By default it's sorted alphabetical ([default Unicode collation](https://www.unicode.org/reports/tr10/)?), but you can enable custom order and drag the order you desire.
